components/header.tsx

```diff
@@ -1,18 +1,72 @@
 import { useLocale } from '@/lib/hooks'
+import { useState } from 'react'
 import OneZeroSkipToMainContent from './onezero-skip-to-main-content'
+import { Button } from './dataComponents/button'
+import { ChangeLangButton } from './dataComponents/changeLangButton'
+import { NextRouter, useRouter } from 'next/router'
 
-export default function Header() {
+export default function Header({ locale }: { locale: string }) {
+  const [headerLinks, setHeader] = useState([
+    'בית',
+    'אודות המרפאה',
+    'צרו קשר',
+    'תמכו בנו',
+  ]) // set by default to heb
+
+  // const hebHeaderLinks = ['בית', 'אודות המרפאה', 'צרו קשר', 'תמכו בנו']
+  const engHeaderLinks = [
```

Comment:
Don't you have those values coming from the CMS?

Reply:
fixed,
now i have those values